FlaskSite/__init__.py

```diff
@@ -10,13 +10,32 @@
 migrate = Migrate()
 
 
+def createDefaultConfig():
+    """
+        This will generate the config.py file if not found
+    """
+    with open("FlaskSite/config.py",'w') as f:
+        f.write("import os\n")
+        f.write("# Default configuration settings\n")
+        f.write("DEBUG = False\n")
+        f.write("SECRET_KEY = 'Make It Hard'\n")
+        f.write("BASE_DIR = os.path.abspath(os.path.dirname(__file__))\n")
+        f.write("SQLALCHEMY_DATABASE_URI = 'sqlite:///' + os.path.join(BASE_DIR, 'instance', 'site.db')\n")
+        f.write("IMG_FOLDER = os.path.join(BASE_DIR, 'instance', 'uploads')\n")
+
+
+
 def createApp():
     app = Flask(__name__)
 
     # Enable CORS (Adjust as needed)
     # CORS(app, resources={r"/*": {"origins": "*"}})
 
     # Load configurations
+    if not os.path.exists('config.py'):
```

Comment:
try using FlaskSite/utils/file_utils.py if you can't find the right method to use create a static method there